modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,59 @@
-        favorites: # configuration for the parent menu item
-          icon: favorite # icon from RHDH system icons
-          title: Favorites # title for the parent menu item
-          priority: 100 # controls the order of this top-level menu item
-----
---
-
-. To ensure that a menu item is identified as a main menu item, you must add the `default.` prefix to its key. For example:
-+
---
-.Example configuration of main menu items in sidebar navigation
-[source,yaml]
-----
-dynamicPlugins:
-  frontend:
-    default.main-menu-items: # key for configuring static main menu items
-      menuItems: 
-        default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
-          parent: my_menu_group # optional, specifies the parent menu item for this item
-          priority: 10 # optional, specifies the order of this menu item within its menu level
-        default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main group parent item key <1>
-          icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
-          title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
-          priority: 100 # optional, specifies the order of the parent menu item in the sidebar
-----
-
-
-<1> The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
-
-[NOTE]
-====
-The default priority of main menu items determines their order in the sidebar. You can customize the order of the static main menu items by adjusting their priority values. Ensure that the priority and title of each item is clear to facilitate easy reordering. 
-====
---
-
-
-
-
-
-
-
+        default._<menu_item_name>_: # <1>
+          parent: _<my_menu_group>_ # <2>
+          priority: 10 # <3>
+        default._<menu_group_parent_item_name>_: # <4>
+          icon:  # home | group | category | extension | school | _<your_icon>_ # <5>
+          title: _<my_menu_group_title>_ # <6>
+          priority: 100 # <7>
+----
+<1> `default._<menu_item_name>_`: Enter the menu item name. Add the `default.` prefix to identify a main menu item.
+<2> `parent`: (Optional) Enter the parent menu item for this item.
+<3> `priority`: (Optional) Enter the order of this menu item within its menu level.
+<4> `default._<menu_group_parent_item_name>_`: Enter the menu group parent item name to configure static main menu items.
+<5> `icon`: Enter the menu group icon. Required for parent menu items.
+<6> `title`: Enter the menu group title. Required for parent menu items. To hide the title from the sidebar, set the title as an (`" "`) empty string.
```

Comment:
```suggestion
<6> `title`: Enter the menu group title. Required for parent menu items. To hide the title from the sidebar, set the title as an (`" "`) empty string.
<7> `to`: (Optional) Enter the path the menu item navigates to. If not set, it defaults to the home page.
```

Note that I didn't adjust the number marks in yaml markdown. This is to show the best place to add `to` explanation.